events/using/aem-event-setup.md

```diff
@@ -26,14 +26,19 @@ Before setting up and using AEM with Adobe I/O, you will need to do the followin
 
 To complete this solution, you will need authorization to use the following services:
 
-*   An AEM instance, version 6.2 or 6.3, with administrative permissions. (**Note:** AEM screens in this topic are captured from version 6.3.)
+*   An AEM instance, version `6.2.x`, `6.3.x` or `6.4.x`, with administrative permissions. (**Note:** AEM screens in this topic are captured from version 6.3.)
 *   [Adobe I/O Console](https://adobe.io/console) access, with administrative permissions for your enterprise organization. If you do not have administrative permissions, please contact [ioevents-beta@adobe.com](mailto:ioevents-beta@adobe.com). After requesting administrative permissions, watch for an email from Adobe Systems Incorporated, notifying you of your permissions:
 
       ![Admin rights email](../../img/events_aem_01.png "Admin rights email")
 
 ### Register an AEM event consumer app
 
-You will need to register an AEM event consumer app, such as a webhook, to see responses to AEM changes. These instructions include steps for setting up a webhook that is able to accept and reply to a [challenge HTTP request](../intro/webhook_docs_intro.md##thechallengerequest) parameter sent by Adobe I/O Channel & Subscription Management (CSM). For more information on understanding and working with webhooks, see the [Introduction to Adobe I/O Events Webhooks](../intro/webhook_docs_intro.md).
+You will need to register an AEM event consumer app, such as a webhook, to see responses to AEM changes. 
+These instructions include steps for setting up a webhook that is able to accept 
+and reply to a [challenge HTTP request](../intro/webhook_docs_intro.md##thechallengerequest) parameter sent 
+by Adobe I/O Channel & Subscription Management (CSM). 
```

Comment:
Please, let's change this, as CSM is an internal implementation detail. Better: "... sent by I/O Events"